README.md

```diff
@@ -84,11 +84,11 @@ In order to optimize support effort,
 
 	On Ubuntu:
 
-		 sudo apt install astyle cmake gcc ninja-build libssl-dev python3-pytest python3-pytest-xdist unzip xsltproc doxygen graphviz python3-yaml valgrind
+		 sudo apt install astyle cmake gcc libssl-dev python3-pytest python3-pytest-xdist unzip xsltproc doxygen graphviz python3-yaml valgrind
 
 	On macOS, using a package manager of your choice (we've picked Homebrew):
 
-		brew install cmake ninja openssl@1.1 wget doxygen graphviz astyle valgrind
+		brew install cmake openssl@1.1 wget doxygen graphviz astyle valgrind
```

Comment:
Yikes -- we still suggest using OpenSSL1.1??

Reply:
oof. Good catch. I'll make another pr updating that to `openssl@3`.